jobs/rocoto/metp.sh

```diff
@@ -51,6 +51,13 @@ export DATAROOT="$RUNDIR/$CDATE/$CDUMP/metp.${jobid}"
 [[ -d $DATAROOT ]] && rm -rf $DATAROOT
 mkdir -p $DATAROOT
 
+if [ $machine = "HERA" -o $machine = "ORION" ]; then
+  unset prod_util_ver
+  unset grib_util_ver
+  unset nco_ver
+  unset intel_ver
+  unset anaconda_ver
```

Comment:
Reminder to myself and note to everyone else: Kate and I discussed this and decided this was the best way to handle things for dev 16, which uses an older version of the metp script that loads its own modules with specific versions that are different than we are using elsewhere. In develop, these will be set properly as we have a verif branch that no longer loads its own modules.